odk1-src/form-logic.rst

```diff
@@ -240,44 +264,71 @@ without answering the question.
 Setting default responses
 ===========================
 
-To provide a default response to a question,
-put the response value in the :th:`default` column.
+To provide a default response to a question, put a value in the :th:`default` column. Defaults are set once when a record is first created from a form definition. Defaults can either be fixed values (:ref:`static defaults <static-defaults>`) or the result of some expression (:ref:`dynamic defaults <dynamic-defaults>`). 
 
-Default values must be static values,
-not expressions or variables.
+.. _static-defaults:
 
-.. note::
+Static defaults
+----------------
 
-  The content of the :th:`default` row in a question
-  is taken literally as the default value.
-  Quotes should **not** be used to wrap string values,
-  unless you actually want those quote marks to appear
-  in the default response value.
+The text in the :th:`default` column for a question is taken literally as the default value. Quotes should **not** be used to wrap values, unless you actually want those quote marks to appear in the default response value.
 
-.. rubric:: XLSForm
+In the example below, the "Phone call" option with underlying value ``phone_call`` will be selected when the question is first displayed. The enumerator can either keep that selection or change it.
+
+.. rubric:: XLSForm to select "Phone call" as the default contact method
 
 .. csv-table:: survey
   :header: type, name, label, default
-  
+
   select_one contacts, contact_method, How should we contact you?, phone_call
-  
+
 .. csv-table:: choices
   :header: list_name, name, label
-  
+
   contacts, phone_call, Phone call
   contacts, text_message, Text message
   contacts, email, Email
+
+.. _dynamic-defaults:
+
+Dynamic defaults
+----------------
+
+.. warning::
   
+  Support for :ref:`dynamic defaults <dynamic-defaults>` was added in Collect v1.24.0. Form conversion requires XLSForm Online ≥ v2.0.0 or pyxform ≥ v1.0.0. Using older versions will have unpredictable results.
+
+If you put an expression in the :th:`default` column for a question, that expression will be evaluated once when a record is first created from a blank form definition. This allows you to use values from outside the form like the current date or the :ref:`server username <metadata>`. Dynamic defaults can't be used to base the default value of one field on the value of another field in the form. For example, if you have an ``age`` question and you want to default the diagnosis age to the value of the ``age`` question, see :ref:`the tip below <defaults-from-form-data>`.
```

Comment:
I still feel like this "once" is hard to understand. It'd be good to break it out. 

s/"used to base"/"used to set"

Reply:
Drop the age question example. You say you can't do something, then you provide an example of what you can do. 

```
field in the form. If you wish to do that, see :ref:`the tip below <defaults-from-form-data>`.